src/main/resources/messages.properties

```diff
@@ -99,4 +99,8 @@ validation.length.maxExceeded.accounting_policies.basis_of_measurement_and_prepa
 
 validation.characters.invalid.accounting_policies.turnover_policy = Enter valid characters
 validation.length.minInvalid.accounting_policies.turnover_policy = Give details of your 'turnover policy'
-validation.length.invalidInputLength.accounting_policies.turnover_policy = You must not exceed {1} characters
\ No newline at end of file
+validation.length.invalidInputLength.accounting_policies.turnover_policy = You must not exceed {1} characters
+
+validation.characters.invalid.accounting_policies.intangible_fixed_assets_amortisation_policy = Enter valid characters
+validation.length.minInvalid.accounting_policies.intangible_fixed_assets_amortisation_policy = Give details of your ''intangible fixed assets amortisation'' policy
```

Comment:
is this supposed to be ' 'intangible fixed assets amortisation' '? with double ' '

Reply:
Yeah, this is a Thymeleaf thing. The first ' escapes the second to the text reads onscreen as 'intangible fixed assets amortisation'